common/src/test/java/org/opensearch/ml/common/utils/StringUtilsTest.java

```diff
@@ -117,6 +118,20 @@ public void getParameterMap() {
         assertEquals("[1.01,\"abc\"]", parameterMap.get("key5"));
     }
 
+    @Test
+    public void getInterfaceMap() {
+        Map<String, Object> parameters = new HashMap<>();
+        parameters.put("input", "value1");
+        parameters.put("output", 2);
+        parameters.put("key3", 2.1);
```

Comment:
so the keys which are not "input" and "output" will be ignored? users can define a model interface but not using "input" and "output" then it would be empty model interface? is it the intention here?

Reply:
Yes you are right